pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -1461,6 +1458,12 @@ func (p *PriorityQueue) getBackoffTime(podInfo *framework.QueuedPodInfo) time.Ti
 // calculateBackoffDuration is a helper function for calculating the backoffDuration
 // based on the number of attempts the pod has made.
 func (p *PriorityQueue) calculateBackoffDuration(podInfo *framework.QueuedPodInfo) time.Duration {
+	if podInfo.Attempts == 0 {
```

Comment:
what's the name of the test for this?

Reply:
`QueueHintFunction is called when Pod is gated by a plugin other than SchedulingGate` test case in `TestPriorityQueue_MoveAllToActiveOrBackoffQueueWithQueueingHint` ensures that the Pod with zero attempt doesn't get backoff.

Reply:
what if the hints are disabled?

Reply:
The test isn't related to the feature gate.
It's just that when the feature gate is disabled we don't accept the queueing hint from the plugin, but still use [the default queueing hint](https://github.com/kubernetes/kubernetes/blob/b5a156971fa21863996cc6ab207d7120b677d2b7/pkg/scheduler/scheduler.go#L384-L389).